coordinator/coordinator.go

```diff
@@ -272,58 +342,91 @@ func (c *Coordinator) runFeatureMaterializeJob(resID metadata.ResourceID) error
 	}
 	status := feature.Status()
 	featureType := feature.Type()
-	if status == metadata.READY || status == metadata.FAILED {
+	if status == metadata.READY { // change this?
```

Comment:
Some way of noting this needs to be changed back once we can un-hack it

Reply:
Made asana task